Scripts/Tracking/Velocity/VelocityTrackerProcessor.cs

```diff
@@ -22,11 +22,20 @@ public override bool IsActive()
             return isActiveAndEnabled;
         }
 
+        /// <summary>
+        /// The current active <see cref="VelocityTracker"/> that is reporting velocities.
+        /// </summary>
+        /// <returns>The current active <see cref="VelocityTracker"/>.</returns>
+        public virtual VelocityTracker GetActiveVelocityTracker()
+        {
+            return (cachedTracker != null && cachedTracker.IsActive() ? cachedTracker : null);
+        }
+
         /// <summary>
         /// The reported velocity on the first active <see cref="VelocityTracker"/>.
         /// </summary>
         /// <returns>The current velocity.</returns>

```

Comment:
This XML docs block should just be `<inheritdoc />`.

Reply:
The docs are actually different for this method, it's doing something specific